lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -67,36 +61,77 @@ export class CdnSiteHostingConstruct extends cdk.Construct {
     });
     new cdk.CfnOutput(this, "Bucket", { value: this.s3Bucket.bucketName });
 
+    const defaultSecurityHeaders: cloudfront.ResponseSecurityHeadersBehavior = {
+      // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src
+      contentSecurityPolicy: {
+        contentSecurityPolicy: "default-src 'self';",
```

Comment:
question: how have we decided on the defaults for all of these headers?

Reply:
I've just taken all the defaults from the recommended from commented link above it.

Reply:
How will this affect authentication in our SPAs?